ERROR - 2024-06-16 05:38:38 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(59): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 05:38:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(59): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:38:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:38:38\', \'2024-06-16 05:38:38\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:38:38\', \'2024-06-16 05:38:38\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(59): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:38:38 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:38:38\', \'2024-06-16 05:38:38\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:38:38\', \'2024-06-16 05:38:38\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:38:38\', \'2024-06-16 05:38:38\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(59): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 05:41:23 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 05:41:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:41:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:41:23\', \'2024-06-16 05:41:23\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:41:23\', \'2024-06-16 05:41:23\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:41:23 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:41:23\', \'2024-06-16 05:41:23\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:41:23\', \'2024-06-16 05:41:23\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:41:23\', \'2024-06-16 05:41:23\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 05:45:33 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 05:45:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:45:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:45:33\', \'2024-06-16 05:45:33\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:45:33\', \'2024-06-16 05:45:33\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:45:33 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:45:33\', \'2024-06-16 05:45:33\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:45:33\', \'2024-06-16 05:45:33\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:45:33\', \'2024-06-16 05:45:33\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 05:49:44 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 05:49:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:49:44 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:49:44\', \'2024-06-16 05:49:44\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:49:44\', \'2024-06-16 05:49:44\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:49:44 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:49:44\', \'2024-06-16 05:49:44\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:49:44\', \'2024-06-16 05:49:44\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:49:44\', \'2024-06-16 05:49:44\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:53:35 --> GuzzleHttp\Exception\ClientException: Client error: `POST https://oauth2.googleapis.com/token` resulted in a `400 Bad Request` response:
{
  "error": "invalid_grant",
  "error_description": "Bad Request"
}

in VENDORPATH\guzzlehttp\guzzle\src\Exception\RequestException.php on line 113.
 1 VENDORPATH\guzzlehttp\guzzle\src\Middleware.php(72): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), null, [], null)
 2 VENDORPATH\guzzlehttp\promises\src\Promise.php(209): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response))
 3 VENDORPATH\guzzlehttp\promises\src\Promise.php(158): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), null)
 4 VENDORPATH\guzzlehttp\promises\src\TaskQueue.php(52): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
 5 VENDORPATH\guzzlehttp\promises\src\Promise.php(251): GuzzleHttp\Promise\TaskQueue->run(true)
 6 VENDORPATH\guzzlehttp\promises\src\Promise.php(227): GuzzleHttp\Promise\Promise->invokeWaitFn()
 7 VENDORPATH\guzzlehttp\promises\src\Promise.php(272): GuzzleHttp\Promise\Promise->waitIfPending()
 8 VENDORPATH\guzzlehttp\promises\src\Promise.php(229): GuzzleHttp\Promise\Promise->invokeWaitList()
 9 VENDORPATH\guzzlehttp\promises\src\Promise.php(69): GuzzleHttp\Promise\Promise->waitIfPending()
10 VENDORPATH\guzzlehttp\guzzle\src\Client.php(124): GuzzleHttp\Promise\Promise->wait()
11 VENDORPATH\google\auth\src\HttpHandler\Guzzle6HttpHandler.php(47): GuzzleHttp\Client->send(Object(GuzzleHttp\Psr7\Request), [...])
12 VENDORPATH\google\auth\src\OAuth2.php(551): Google\Auth\HttpHandler\Guzzle6HttpHandler->__invoke(Object(GuzzleHttp\Psr7\Request))
13 VENDORPATH\google\apiclient\src\Google\Client.php(215): Google\Auth\OAuth2->fetchAuthToken(Object(Google\Auth\HttpHandler\Guzzle7HttpHandler))
14 APPPATH\Controllers\Peternak.php(40): Google_Client->fetchAccessTokenWithAuthCode('4/0ATx3LY58lO1gANx-SxU8_YF6DPHTuELAM5_qRV4Q2cVtkKyDisUtsmbAHqhSifF-UOk8ZA')
15 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
16 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
17 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 05:59:25 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 05:59:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:59:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:59:25\', \'2024-06-16 05:59:25\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:59:25\', \'2024-06-16 05:59:25\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 05:59:25 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:59:25\', \'2024-06-16 05:59:25\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:59:25\', \'2024-06-16 05:59:25\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 05:59:25\', \'2024-06-16 05:59:25\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:05:11 --> ErrorException: Can't use function return value in write context
in APPPATH\Controllers\Front.php on line 10.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-06-16 06:07:28 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:07:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:07:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:07:28\', \'2024-06-16 06:07:28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:07:28\', \'2024-06-16 06:07:28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:07:28 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:07:28\', \'2024-06-16 06:07:28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:07:28\', \'2024-06-16 06:07:28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:07:28\', \'2024-06-16 06:07:28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(58): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:18:34 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:18:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:18:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:18:34\', \'2024-06-16 06:18:34\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:18:34\', \'2024-06-16 06:18:34\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:18:34 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:18:34\', \'2024-06-16 06:18:34\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:18:34\', \'2024-06-16 06:18:34\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:18:34\', \'2024-06-16 06:18:34\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:19:44 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:19:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:19:44 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:19:44\', \'2024-06-16 06:19:44\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:19:44\', \'2024-06-16 06:19:44\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:19:44 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:19:44\', \'2024-06-16 06:19:44\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:19:44\', \'2024-06-16 06:19:44\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:19:44\', \'2024-06-16 06:19:44\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:20:31 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\Peternak.php(58): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:21:08 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:21:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:21:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:21:08\', \'2024-06-16 06:21:08\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:21:08\', \'2024-06-16 06:21:08\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:21:08 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:21:08\', \'2024-06-16 06:21:08\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:21:08\', \'2024-06-16 06:21:08\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:21:08\', \'2024-06-16 06:21:08\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:25:13 --> ErrorException: Trying to access array offset on value of type int
in APPPATH\Controllers\Peternak.php on line 51.
 1 APPPATH\Controllers\Peternak.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type int', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Peternak.php', 51)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:26:26 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\Peternak.php(58): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:31:49 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:31:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:31:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:31:49\', \'2024-06-16 06:31:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:31:49\', \'2024-06-16 06:31:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:31:49 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:31:49\', \'2024-06-16 06:31:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:31:49\', \'2024-06-16 06:31:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:31:49\', \'2024-06-16 06:31:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:32:16 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:32:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:32:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:32:16\', \'2024-06-16 06:32:16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:32:16\', \'2024-06-16 06:32:16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:32:16 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:32:16\', \'2024-06-16 06:32:16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:32:16\', \'2024-06-16 06:32:16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:32:16\', \'2024-06-16 06:32:16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 06:34:27 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\sobatx\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\sobatx\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\sobatx\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#10 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#11 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-16 06:34:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:34:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:34:27\', \'2024-06-16 06:34:27\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:34:27\', \'2024-06-16 06:34:27\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:34:27 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:34:27\', \'2024-06-16 06:34:27\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:34:27\', \'2024-06-16 06:34:27\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\', \'2024-06-16 06:34:27\', \'2024-06-16 06:34:27\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`, `created_at`, `updated_at`) VALUES (:id_user.1:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Peternak.php(75): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:35:57 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\Peternak.php(58): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:37:58 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 58 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(58): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:39:26 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 58 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(58): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:40:57 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 57 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(57): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:43:43 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 57 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(57): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:44:58 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 57 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(57): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:46:30 --> GuzzleHttp\Exception\ClientException: Client error: `POST https://oauth2.googleapis.com/token` resulted in a `400 Bad Request` response:
{
  "error": "invalid_grant",
  "error_description": "Bad Request"
}

in VENDORPATH\guzzlehttp\guzzle\src\Exception\RequestException.php on line 113.
 1 VENDORPATH\guzzlehttp\guzzle\src\Middleware.php(72): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), null, [], null)
 2 VENDORPATH\guzzlehttp\promises\src\Promise.php(209): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response))
 3 VENDORPATH\guzzlehttp\promises\src\Promise.php(158): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), null)
 4 VENDORPATH\guzzlehttp\promises\src\TaskQueue.php(52): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
 5 VENDORPATH\guzzlehttp\promises\src\Promise.php(251): GuzzleHttp\Promise\TaskQueue->run(true)
 6 VENDORPATH\guzzlehttp\promises\src\Promise.php(227): GuzzleHttp\Promise\Promise->invokeWaitFn()
 7 VENDORPATH\guzzlehttp\promises\src\Promise.php(272): GuzzleHttp\Promise\Promise->waitIfPending()
 8 VENDORPATH\guzzlehttp\promises\src\Promise.php(229): GuzzleHttp\Promise\Promise->invokeWaitList()
 9 VENDORPATH\guzzlehttp\promises\src\Promise.php(69): GuzzleHttp\Promise\Promise->waitIfPending()
10 VENDORPATH\guzzlehttp\guzzle\src\Client.php(124): GuzzleHttp\Promise\Promise->wait()
11 VENDORPATH\google\auth\src\HttpHandler\Guzzle6HttpHandler.php(47): GuzzleHttp\Client->send(Object(GuzzleHttp\Psr7\Request), [...])
12 VENDORPATH\google\auth\src\OAuth2.php(551): Google\Auth\HttpHandler\Guzzle6HttpHandler->__invoke(Object(GuzzleHttp\Psr7\Request))
13 VENDORPATH\google\apiclient\src\Google\Client.php(215): Google\Auth\OAuth2->fetchAuthToken(Object(Google\Auth\HttpHandler\Guzzle7HttpHandler))
14 APPPATH\Controllers\Peternak.php(41): Google_Client->fetchAccessTokenWithAuthCode('4/0ATx3LY58Pg2w57VuTuhgDupCKTTU9iBm3lqaGqz1f7oguAIrkr84kw8y0f5EfNnAC4IMIA')
15 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
16 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
17 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:48:27 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 57 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(57): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:50:32 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 58 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(58): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 06:50:57 --> GuzzleHttp\Exception\ClientException: Client error: `POST https://oauth2.googleapis.com/token` resulted in a `400 Bad Request` response:
{
  "error": "invalid_grant",
  "error_description": "Bad Request"
}

in VENDORPATH\guzzlehttp\guzzle\src\Exception\RequestException.php on line 113.
 1 VENDORPATH\guzzlehttp\guzzle\src\Middleware.php(72): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), null, [], null)
 2 VENDORPATH\guzzlehttp\promises\src\Promise.php(209): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response))
 3 VENDORPATH\guzzlehttp\promises\src\Promise.php(158): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), null)
 4 VENDORPATH\guzzlehttp\promises\src\TaskQueue.php(52): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
 5 VENDORPATH\guzzlehttp\promises\src\Promise.php(251): GuzzleHttp\Promise\TaskQueue->run(true)
 6 VENDORPATH\guzzlehttp\promises\src\Promise.php(227): GuzzleHttp\Promise\Promise->invokeWaitFn()
 7 VENDORPATH\guzzlehttp\promises\src\Promise.php(272): GuzzleHttp\Promise\Promise->waitIfPending()
 8 VENDORPATH\guzzlehttp\promises\src\Promise.php(229): GuzzleHttp\Promise\Promise->invokeWaitList()
 9 VENDORPATH\guzzlehttp\promises\src\Promise.php(69): GuzzleHttp\Promise\Promise->waitIfPending()
10 VENDORPATH\guzzlehttp\guzzle\src\Client.php(124): GuzzleHttp\Promise\Promise->wait()
11 VENDORPATH\google\auth\src\HttpHandler\Guzzle6HttpHandler.php(47): GuzzleHttp\Client->send(Object(GuzzleHttp\Psr7\Request), [...])
12 VENDORPATH\google\auth\src\OAuth2.php(551): Google\Auth\HttpHandler\Guzzle6HttpHandler->__invoke(Object(GuzzleHttp\Psr7\Request))
13 VENDORPATH\google\apiclient\src\Google\Client.php(215): Google\Auth\OAuth2->fetchAuthToken(Object(Google\Auth\HttpHandler\Guzzle7HttpHandler))
14 APPPATH\Controllers\Peternak.php(42): Google_Client->fetchAccessTokenWithAuthCode('4/0ATx3LY6f9byXme4qMp-lPPvoLyZFbAWRo8AQQ039lnHFZa7_0IFmxMuy_vMp_lP8P8yCDg')
15 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
16 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
17 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:17:42 --> ArgumentCountError: Too few arguments to function App\Models\M_User::updateDataUser(), 1 passed in C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php on line 58 and exactly 2 expected
in APPPATH\Models\M_User.php on line 32.
 1 APPPATH\Controllers\Peternak.php(58): App\Models\M_User->updateDataUser([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 07:22:55 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\app\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(64): App\Models\M_User->saveDataUser(Array)
#6 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#7 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#8 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-16 07:22:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 2 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Peternak.php(64): App\Models\M_User->saveDataUser([...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:22:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 4 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Peternak.php(64): App\Models\M_User->saveDataUser([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:22:55 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 5 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Peternak.php(64): App\Models\M_User->saveDataUser([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:24:44 --> Error: Undefined constant "App\Controllers\id"
in APPPATH\Controllers\Peternak.php on line 48.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 07:27:32 --> mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\app\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser(Array)
#6 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#7 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#8 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-16 07:27:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 2 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:27:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 4 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:27:32 --> [Caused by] mysqli_sql_exception: Duplicate entry '1147168220' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'114716822027796441665\', \'Ferdy Rahmat\', \'ferdyyrahmat@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocL1pTOLtcAPAPp5h89mD753TajbrnlB3bopQxEglUEdcGSKk4o_=s96-c\', \'1\', \'1\', \'0\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 5 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
ERROR - 2024-06-16 07:30:11 --> mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY' in C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\sobatx\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\sobatx\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\sobatx\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\sobatx\app\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\sobatx\app\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser(Array)
#6 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
#7 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
#8 C:\xampp\htdocs\sobatx\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\sobatx\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\sobatx\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-16 07:30:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 2 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:30:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 4 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:30:11 --> [Caused by] mysqli_sql_exception: Duplicate entry '1111154430' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (\'111115443078833748165\', \'SobatX\', \'sobatx.mail@gmail.com\', \'\', \'\', \'\', \'https://lh3.googleusercontent.com/a/ACg8ocJYsmIejUvJJLayNIX0PvDyVwWRKP43QOP8zQaDluTCEmRapw=s96-c\', \'1\', \'1\', \'0\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`id_user`, `nama_user`, `email_user`, `alamat_user`, `no_hp`, `password`, `foto_user`, `akses_level`, `status`, `is_delete_user`) VALUES (:id_user:, :nama_user:, :email_user:, :alamat_user:, :no_hp:, :password:, :foto_user:, :akses_level:, :status:, :is_delete_user:)', [...], false)
 5 APPPATH\Models\M_User.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Peternak.php(65): App\Models\M_User->saveDataUser([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:34:51 --> ErrorException: Trying to access array offset on value of type null
in APPPATH\Controllers\Peternak.php on line 52.
 1 APPPATH\Controllers\Peternak.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Peternak.php', 52)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:37:02 --> GuzzleHttp\Exception\ClientException: Client error: `POST https://oauth2.googleapis.com/token` resulted in a `400 Bad Request` response:
{
  "error": "invalid_grant",
  "error_description": "Bad Request"
}

in VENDORPATH\guzzlehttp\guzzle\src\Exception\RequestException.php on line 113.
 1 VENDORPATH\guzzlehttp\guzzle\src\Middleware.php(72): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), null, [], null)
 2 VENDORPATH\guzzlehttp\promises\src\Promise.php(209): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response))
 3 VENDORPATH\guzzlehttp\promises\src\Promise.php(158): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), null)
 4 VENDORPATH\guzzlehttp\promises\src\TaskQueue.php(52): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
 5 VENDORPATH\guzzlehttp\promises\src\Promise.php(251): GuzzleHttp\Promise\TaskQueue->run(true)
 6 VENDORPATH\guzzlehttp\promises\src\Promise.php(227): GuzzleHttp\Promise\Promise->invokeWaitFn()
 7 VENDORPATH\guzzlehttp\promises\src\Promise.php(272): GuzzleHttp\Promise\Promise->waitIfPending()
 8 VENDORPATH\guzzlehttp\promises\src\Promise.php(229): GuzzleHttp\Promise\Promise->invokeWaitList()
 9 VENDORPATH\guzzlehttp\promises\src\Promise.php(69): GuzzleHttp\Promise\Promise->waitIfPending()
10 VENDORPATH\guzzlehttp\guzzle\src\Client.php(124): GuzzleHttp\Promise\Promise->wait()
11 VENDORPATH\google\auth\src\HttpHandler\Guzzle6HttpHandler.php(47): GuzzleHttp\Client->send(Object(GuzzleHttp\Psr7\Request), [...])
12 VENDORPATH\google\auth\src\OAuth2.php(551): Google\Auth\HttpHandler\Guzzle6HttpHandler->__invoke(Object(GuzzleHttp\Psr7\Request))
13 VENDORPATH\google\apiclient\src\Google\Client.php(215): Google\Auth\OAuth2->fetchAuthToken(Object(Google\Auth\HttpHandler\Guzzle7HttpHandler))
14 APPPATH\Controllers\Peternak.php(42): Google_Client->fetchAccessTokenWithAuthCode('4/0ATx3LY7xUK6PZCyoQ00UyB-MJBlyVcxAp2VHwLQncBLsIX1x8ZyJ9A0klKs7yqW_ahGi8A')
15 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
16 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
17 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:39:14 --> ErrorException: Trying to access array offset on value of type null
in APPPATH\Controllers\Peternak.php on line 52.
 1 APPPATH\Controllers\Peternak.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Peternak.php', 52)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Peternak->login_checker()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Peternak))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:49:45 --> ErrorException: Undefined array key "
            error "
in APPPATH\Views\Frontend\Login\user_page.php on line 144.
 1 APPPATH\Views\Frontend\Login\user_page.php(144): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "
            error "', 'C:\\xampp\\htdocs\\sobatx\\app\\Views\\Frontend\\Login\\user_page.php', 144)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\sobatx\\app\\Views\\Frontend\\Login\\user_page.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('Frontend/Login/user_page', [], true)
 5 APPPATH\Controllers\Front.php(24): view('Frontend/Login/user_page')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->user_select()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 07:58:59 --> ErrorException: Undefined variable $link
in APPPATH\Views\Frontend\Register\register.php on line 80.
 1 APPPATH\Views\Frontend\Register\register.php(80): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $link', 'C:\\xampp\\htdocs\\sobatx\\app\\Views\\Frontend\\Register\\register.php', 80)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\sobatx\\app\\Views\\Frontend\\Register\\register.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('Frontend/Register/register', [], true)
 5 APPPATH\Controllers\Front.php(34): view('Frontend/Register/register')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->register_page()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:31:03 --> ErrorException: Undefined variable $data
in APPPATH\Controllers\Front.php on line 36.
 1 APPPATH\Controllers\Front.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 36)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->register_page()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:32:24 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Frontend/Register/regiter-peternak.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Frontend/Register/regiter-peternak.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('Frontend/Register/regiter-peternak', [], true)
 3 APPPATH\Controllers\Front.php(42): view('Frontend/Register/regiter-peternak', [...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->register_peternak()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:40:56 --> ErrorException: Undefined variable $uri
in APPPATH\Controllers\Front.php on line 17.
 1 APPPATH\Controllers\Front.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $uri', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Front->__construct()
 3 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:40:59 --> ErrorException: Undefined variable $uri
in APPPATH\Controllers\Front.php on line 17.
 1 APPPATH\Controllers\Front.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $uri', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Front->__construct()
 3 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:41:10 --> ErrorException: Undefined variable $uri
in APPPATH\Controllers\Front.php on line 17.
 1 APPPATH\Controllers\Front.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $uri', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Front->__construct()
 3 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:41:57 --> InvalidArgumentException: Redirect URI must be absolute
in VENDORPATH\google\auth\src\OAuth2.php on line 805.
 1 VENDORPATH\google\auth\src\OAuth2.php(363): Google\Auth\OAuth2->setRedirectUri('')
 2 VENDORPATH\google\apiclient\src\Google\Client.php(1012): Google\Auth\OAuth2->__construct([...])
 3 VENDORPATH\google\apiclient\src\Google\Client.php(1001): Google_Client->createOAuth2Service()
 4 VENDORPATH\google\apiclient\src\Google\Client.php(362): Google_Client->getOAuth2Service()
 5 APPPATH\Controllers\Front.php(45): Google_Client->createAuthUrl()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->register_peternak()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:43:03 --> ErrorException: Undefined property: App\Controllers\Front::$googleClient
in APPPATH\Controllers\Front.php on line 33.
 1 APPPATH\Controllers\Front.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Front::$googleClient', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->register_peternak()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
CRITICAL - 2024-06-16 09:56:47 --> ErrorException: Undefined property: App\Controllers\Front::$user
in APPPATH\Controllers\Front.php on line 73.
 1 APPPATH\Controllers\Front.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Front::$user', 'C:\\xampp\\htdocs\\sobatx\\app\\Controllers\\Front.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Front->registering_account_peternak()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Front))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\sobatx\\public\\index.php')
